tests/cache/tests.py

```diff
@@ -1155,6 +1155,20 @@ def test_incr_decr_timeout(self):
 
 class BaseMemcachedTests(BaseCacheTests):
 
+    def test_location_parsing(self):
+        locations = [
+            ['server1.tld', 'server2:11211'],
+            'server1.tld;server2:11211',
+            'server1.tld,server2:11211',
+        ]
+        for location in locations:
+            with self.settings(CACHES={
```

Comment:
Perhaps something like this would avoid the interesting indentation:
```
params = {'BACKEND': self.base_params['BACKEND'], 'LOCATION': location}
with self.settings(CACHES={'default': params}):
```